core/templates/components/common-layout-directives/common-elements/sharing-links.directive.ts

```diff
@@ -85,7 +85,7 @@ angular.module('oppia').directive('sharingLinks', [
                 DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR));
 
             ctrl.classroomUrl = UrlInterpolationService.getStaticImageUrl(
-              '/general/classroom.png');
+              '/general/classroom.webp');
```

Comment:
Could we support both png and WebP here?

Reply:
I have made them as picture tags in the directive.html file

Reply:
No, you didn't, I don't see any changes to `sharing-links.directive.hmtl`

Reply:
Also, why do we need this variable then if you set it up in the HTML file.

Reply:
done!